spec/features/mailers/01_email_review_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
```

Comment:
These two feature tests are very similar. I think it would be a good idea to combine them, unless there is some context I'm not aware of. 😎